typescript/web/src/components/export-button/export-modal/export-modal.fixtures.ts

```diff
@@ -0,0 +1,84 @@
+import { ExportFormat } from "../../../graphql-types/globalTypes";
+import {
+  ApolloMockResponse,
+  ApolloMockResponses,
+} from "../../../utils/tests/apollo-mock";
+import { BASIC_DATASET_DATA } from "../../../utils/tests/data.fixtures";
+import { EXPORT_DATASET_URL_QUERY } from "./export-dataset";
+import { COUNT_LABELS_OF_DATASET_QUERY } from "./export-modal.context";
+import { getDatasetName } from "./utils/get-dataset-name";
+import {
+  CountLabelsOfDatasetQueryVariables,
+  CountLabelsOfDatasetQuery,
+} from "../../../graphql-types/CountLabelsOfDatasetQuery";
+import {
+  ExportDatasetUrlQueryVariables,
+  ExportDatasetUrlQuery,
+} from "../../../graphql-types/ExportDatasetUrlQuery";
+
+const datasetName = getDatasetName({
+  datasetSlug: BASIC_DATASET_DATA.slug,
+  format: ExportFormat.COCO,
+});
+
+const EXPORT_DATASET_URL_MOCK: ApolloMockResponse<
+  ExportDatasetUrlQuery,
+  ExportDatasetUrlQueryVariables
+> = {
+  request: {
+    query: EXPORT_DATASET_URL_QUERY,
+    variables: {
+      datasetId: BASIC_DATASET_DATA.id,
+      format: ExportFormat.COCO,
+      options: {
+        coco: {
+          exportImages: false,
+          name: datasetName,
+        },
+        yolo: {
+          exportImages: false,
+          includePolygons: false,
+          name: datasetName,
```

Comment:
`datasetName` has been set with `ExportFormat.COCO`. Isn't there a problem here?
Do you really need to use the dataset name in COCO and YOLO export options?

Reply:
That's how it's used in the mutation, I've reproduced the same behavior

Reply:
OK for the second question. I've written this comment before reading the parts below in the diff 😉 
But what about the first question? The YOLO name should be build with `ExportFormat.YOLO` no?

Reply:
Honestly, I don't know. It does seems weird, and the opposite happens also as it is the same variable used for the both options.

I guess the Yolo options should not be passed when the export format is Coco and vice versa